### DISCUSSION TITLE
Document type and others

### DISCUSSION THREAD

**Discussion by Martafct** [joaomlourenco/novathesis#387] [Q&A] [answered]
Good afternoon. Hope you are doing okay. 

I just wanted to ask if I'm missing something, because im doing the changes on the "Config - 1_nova thesis.tex" and it does not change anything. Not the main language, not the document type, nothing. I sent you two screenshots to show you where I'm changing the configurations. And yes, I did compile the document (thousands of times). 
[
<img width="1326" alt="Captura de ecrã 2022-11-24, às 19 51 07" src="https://user-images.githubusercontent.com/115926172/203856703-50c650a3-1479-42fd-ab33-82dfbcb900d1.png">
<img width="1326" alt="Captura de ecrã 2022-11-24, às 19 51 27" src="https://user-images.githubusercontent.com/115926172/203856710-570028bb-d115-4ec3-a149-1f80e3e21c6c.png">
](url)

**Comment by joaomlourenco** [comment, marked as answer]
You have to _uncomment_ the lines you are changing.
Please learn the basics of LaTeX before using it…. 😱
